container/config.py

```diff
@@ -106,7 +116,7 @@ def set_env(self, env):
                         config['volumes'][vol_key] = docker_settings
             elif self.engine_name in ('openshift', 'k8s'):
                 # For openshift/k8s remove unrelated attributes
-                for vol_key in config['volumes'].keys():
+                for vol_key in list(config['volumes'].keys()):
```

Comment:
Isn't the py3 way of doing this: `for vol_key in config['volumes']:`? Why the cast?